plugins/parsers/json/parser.go

```diff
@@ -126,7 +177,7 @@ func (p *JSONParser) ParseLine(line string) (telegraf.Metric, error) {
 	}
 
 	if len(metrics) < 1 {
-		return nil, fmt.Errorf("Can not parse the line: %s, for data format: influx ", line)
+		return nil, fmt.Errorf("can not parse the line: %s, for data format: influx ", line)
```

Comment:
Also the copypasta, `s/influx/json`

Reply:
could you clarify what you mean here?

Reply:
Replace "influx" with "json", it appears to be copy paste code

Reply:
ah i see. good catch thanks